templates/content_based.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Content Based Movies</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}" />
```

Comment:
It seems that the URL for the stylesheet is being generated dynamically using Flask's url_for function. Make sure that this code is executed in a Flask context, otherwise the stylesheet URL will not be resolved correctly.